language/stdlib/modules/lcs.move

```diff
@@ -0,0 +1,7 @@
+// Utility for serializing a Move type to its binary LCS representation
```

Comment:
Probably want to explain what LCS is, or at least point to further reading elsewhere